modules/apps/frontend-js/frontend-js-react-web/npmscripts.config.js

```diff
@@ -14,9 +14,10 @@
 
 module.exports = {
 	build: {
+		babel: false,
```

Comment:
Would it be possible to just use a `webpack: true` config and then default babel, bundler, and tsc all to false? Or is there a case where we would use both webpack and the others?

Reply:
I've added an `esm` switch in the last commit, I think, but I'm constantly finding reasons to change this over and over again as I progress, so we'll see how this ends...

Ideally, we should have one single switch for the mid term, then remove it when everything is migrated.